Description: Add code to Exec key, remove Encoding key, add Keywords key.
Author: Yavor Doganov <yavor@gnu.org>
Forwarded: gnomint-devel@lists.sourceforge.net
Last-Update: 2018-09-01
---

--- gnomint-1.3.0.orig/gui/gnomint.desktop.in
+++ gnomint-1.3.0/gui/gnomint.desktop.in
@@ -2,11 +2,11 @@
 _Name=gnoMint X.509 CA Manager
 GenericName=gnoMint X.509 CA Manager
 _Comment=Manage X.509 certificates and CAs, easily and graphically
-Exec=gnomint
+_Keywords=certificate;x.509;encryption;
+Exec=gnomint %f
 Terminal=false
 Type=Application
 Icon=gnomint
 Categories=GNOME;Application;System;Security
 MimeType=application/x-gnomint;
 StartupNotify=true
-Encoding=UTF-8
